src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User test = UserService.findByEmail(email);
+        if (test != null) {
+            if (password.equals(test.getPassword())) {
+                return true;
+            }
+        }
         return false;
```

Comment:
Let's simplify it this way:
```suggestion
        return user != null && user.getPassword().equals(password);
```